wazo_lib_rest_client/client.py

```diff
@@ -106,22 +105,21 @@ def session(self):
         if self._token_id:
             session.headers['X-Auth-Token'] = self._token_id
 
-        if self._tenant_id:
-            session.headers['Wazo-Tenant'] = self._tenant_id
+        if self.tenant_uuid:
+            session.headers['Wazo-Tenant'] = self.tenant_uuid
 
         if self._user_agent:
             session.headers['User-agent'] = self._user_agent
 
         return session
 
-    def set_tenant(self, tenant):
-        self._tenant_id = tenant
-
-    def set_default_tenant(self, tenant):
-        self._default_tenant_id = tenant
+    def set_tenant(self, tenant_uuid):
+        logger.warning('set_tenant() is deprecated. Please use tenant_uuid')
```

Comment:
```suggestion
        warning.warn('set_tenant() is deprecated. Please use tenant_uuid', DeprecationWarning)
```
See [the warnings module](https://docs.python.org/3/library/warnings.html) for more details.

Reply:
As discussed IRL,
We will avoid `warning.warn` for now, because to see this warning in our services, we would need to add `logging.captureWarnings(True)` which will also capture warning from other libraries and may cause a lot of unwanted warning